src/main/java/com/worldwarofants/game/ConsoleGame.java

```diff
@@ -48,12 +48,13 @@ public void run() {
 
     @Override
     protected AbstractModule defineStartingModule(IModuleNavigator navigator) {
-        return new ExampleModule(world, navigator);
+        return new StartMenuModule(world, navigator);
     }
 
     @Override
     protected void defineModules(IModuleNavigator navigator) {
-        addModule(new ExampleModule(world, navigator));
+        addModule(new NewGameModule(world, navigator));
+        addModule(new StartMenuModule(world, navigator));
```

Comment:
I think you don't have to define the module which is also the starting module. Check the documentation if it added it on its own. I have a vague memory that it did.

Reply:
Duplication of adding start module removed.